omnibus-sepolia.toml

```diff
@@ -1,5 +1,5 @@
 name = "synthetix-omnibus"
-version = "3.3.4"
+version = "3.3.4-1"
```

Comment:
I can see in docs update PR snx pool was removed (looks like we haven't deployed sepolia  for a long time)

https://github.com/Synthetixio/Synthetix-Gitbook-v3/pull/121/files